cpp/src/io/orc/reader_impl.cu

```diff
@@ -722,15 +722,13 @@ struct list_buffer_data {
 };
 
 // Generates offsets for list buffer from number of elements in a row.
-void generate_offsets_for_list(device_span<list_buffer_data const> buff_data,
+void generate_offsets_for_list(host_span<list_buffer_data> const& buff_data,
```

Comment:
A span can be copied by value.
```suggestion
void generate_offsets_for_list(host_span<list_buffer_data const> buff_data,
```

Reply:
Good point, however the change to the type qualifier here isn't quite right. The underlying list buffers will be changed by the algorithm, it's only the top-level reference that is const. I can accept the whole thing by value though and that should work fine.

Reply:
Changed.